interfaces/builtin/location_control.go

```diff
@@ -68,6 +68,27 @@ dbus (receive)
     member="AddProvider"
     peer=(label=###PLUG_SECURITY_TAGS###),
 
+dbus (send)
+    bus=system
+    path=/providers/{,/**}
```

Comment:
This has an extra '/' that is unneeded and should be rewritten as:
```
path=/providers/{,**}
```

That evaluates to `path=/providers/` and `path=/providers/**` whereas your rule evaluates the second to `path=/providers//**`. It isn't a big deal, but please clean here and elsewhere.